aeon_internal.proto

```diff
@@ -44,3 +51,68 @@ message GetConfigResponse {
   // The gRPC server sideservice configuration.
   Config config = 2;
 }
+
+// Find space by name in space cache.
+
+message FindSpaceRequest {
+  // Name of the space to find.
+  string name = 1;
+}
+
+message FindSpaceResponse {
+  // Error information. Set only on failure.
+  Error error = 1;
+  // Definition of the found space.
+  SpaceDef space = 2;
+}
+
+// Find range by space name and a key in range cache.
+
+// Description of a range.
+message Range {
+  // The range ID.
+  string id = 1;
```

Comment:
Let's order and name the fields exactly as in the Lua router range cache:
- Swap `id` and `shard`.
- Rename `begin` and `end` to `key_begin` and `key_end`.

Also, better define `Range` in the beginning of the file because we may add all ranges to `FindSpaceResponse` in future.